releases/2.x.md

```diff
@@ -1,5 +1,20 @@
 # Waffle Takeout 2.4
 
+## Waffle Takeout 2.4.4
+_Released 2016.08.01_
+
+_Supported GH:E versions: GH:E 2.2+_
+
+#### Features
+
+- More intuitive setup for GitHub Enterprise and GitHub.com integrations.
```

Comment:
either that or retroactively add it to 2.4.3? Sorry, that was my bad for not adding it. here is probably fine too. I'm pretty sure no one will really complain.